src/sentry/api/endpoints/project_transfer.py

```diff
@@ -59,10 +60,21 @@ def post(self, request: Request, project) -> Response:
                 user__email__iexact=email, role=roles.get_top_dog().id, user__is_active=True
             )[0]
         except IndexError:
-            return Response(
-                {"detail": "Could not find an organization owner with that email"},
-                status=status.HTTP_404_NOT_FOUND,
-            )
+            try:
+                org_members = list(
+                    OrganizationMember.objects.filter(
+                        user__email__iexact=email, user__is_active=True
+                    ).values_list("id", flat=True)
+                )
+                org_member_id = OrganizationMemberTeam.objects.filter(
```

Comment:
how does this work exactly? so someone could be an org level "member", but then be on a team that is owner? i searched briefly for docs in the code and didn't see anything, mind pointing me somewhere?

Reply:
yes, that's right. you can be on a team that has an org role that extends the scopes of your user-set organization role https://github.com/getsentry/sentry/blob/master/src/sentry/models/organizationmember.py#L398